testapi.pm

```diff
@@ -434,7 +434,7 @@ if no string is given
 sub type_password {
     my ($string) = @_;
     $string //= $password;
-    type_string $string, secret => 1;
+    type_string $string, 3, secret => 1;
```

Comment:
Are you sure this works? Mixing new and old syntax.
I think this should be like
```perl
type_string $string, max_interval => 3, secret => 1;
```